misk/src/main/kotlin/misk/web/jetty/WebActionsServlet.kt

```diff
@@ -142,10 +145,12 @@ internal class WebActionsServlet @Inject constructor(
       val requestContentType = httpCall.contentType()
       val requestAccepts = httpCall.accepts()
 
-      val candidateActions = boundActions.mapNotNull {
-        it.match(httpCall.dispatchMechanism, requestContentType, requestAccepts, httpCall.url)
+      var bestAction = findBestAction(httpCall.dispatchMechanism, httpCall.url, requestContentType, requestAccepts)
+
+      // If no actions were found, allow POST actions to be considered for handling grpc requests
```

Comment:
I think is worth mentioning this makes sense when You have gRPC calls and you want to handle it manually without using the misk abstraction on top of it.